topcoffea/cfg/xsec.cfg

```diff
@@ -29,6 +29,10 @@ WG                      : 178.8
 TTG                     : 3.697
 TTGJets                 : 3.697
 
+# PLACEHOLDERS!!! NEED TO UPDATE!!!
+TTGamma_SingleLept : 3.697
```

Comment:
We should probably just scale these by similar (but slightly more accurate) calculations to what @klannon did to estimate the number of semileptonic and dileptonic events in our old `TTG` sample, unless we can find explicit ones for these.

Reply:
I've updated the numbers as we discussed on MM.